packages/react-charting/src/components/GaugeChart/GaugeChart.types.ts

```diff
@@ -0,0 +1,252 @@
+import { IStyle, ITheme } from '@fluentui/react/lib/Styling';
+import { IStyleFunctionOrObject } from '@fluentui/react/lib/Utilities';
+import { ILegendsProps } from '../Legends/index';
+import { IAccessibilityProps } from '../../types/index';
+import { ICalloutProps } from '@fluentui/react/lib/Callout';
+
+export interface IGaugeChartSegment {
+  /**
+   * Legend text for a segment
+   */
+  legend: string;
+
+  /**
+   * Size of the segment
+   */
+  size: number;
+
+  /**
+   * Color of the segment. If not provided, a placeholder color will be used.
+   */
+  color?: string;
+
+  /**
+   * Accessibility data for the segment
+   */
+  accessibilityData?: IAccessibilityProps;
+}
+
+export enum GaugeValueFormat {
+  Percentage = 'percentage',
+  Fraction = 'fraction',
+}
+
+export interface IGaugeChartProps {
+  /**
+   * Width of the chart
+   */
+  width?: number;
+
+  /**
+   * Height of the chart
+   */
+  height?: number;
+
+  /**
+   * Title of the chart
+   */
+  chartTitle?: string;
+
+  /**
+   * Current value of the gauge
+   */
+  chartValue: number;
+
+  /**
+   * Sections of the gauge
+   */
+  segments: IGaugeChartSegment[];
+
+  /**
+   * Minimum value of the gauge
+   */
+  minValue?: number;
+
+  /**
+   * Maximum value of the gauge
+   */
+  maxValue?: number;
+
+  /**
+   * Additional text to display below the chart value
+   */
+  sublabel?: string;
+
+  /**
+   * Hide the min and max values of the gauge
+   * @defaultvalue false
+   */
+  hideLimits?: boolean;
```

Comment:
We can call it hideMinMax for readability.

Reply:
done